packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -136,7 +136,7 @@ const useBigNumberConfig = (
 
     const [selectedField, setSelectedField] = useState<string | undefined>();
     const getField = useCallback(
-        (fieldNameOrId: string) => {
+        (fieldNameOrId: string | undefined) => {
```

Comment:
highly unlikely but wonder if this change might cause a bug.
just to be safe we add this guard in the function `if (fieldNameOrId === undefined) return`

Reply:
I'm happy to do that. But I think this will return undefined when passed undefined anyway, no?

Reply:
`isField` will handle properly but not sure about the second condition `.name === fieldNameOrId`

Reply:
Ok, I added the guard. Honestly I like to use unit tests for things like this. I might add some.